### PR TITLE
Rewrite the Chart - Points Aggregation demo description(T929184)

### PR DESCRIPTION
**Pull request by AlexJameson**

### CODE REVIEW

File: JSDemos/Demos/Charts/PointsAggregationFinancialChart/description.md [hunk 1]
```diff
@@ -1 +1,5 @@
-This demo illustrates how the **Chart** widget aggregates series points in&nbsp;a&nbsp;financial chart. The **useAggregation** option activates this feature. The **RangeSelector** widget zooms the contents of&nbsp;the **Chart**. Note that whenever the selected range is&nbsp;reduced, the series becomes less aggregated.
\ No newline at end of file
+The **Chart** widget can aggregate data represented by series points. To enable this feature, set the **aggregation**.[enabled](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/series/aggregation/#enabled) option of a [series](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/series/) to **true**.
```
Comment (RomanTsukanov) [inline JSDemos/Demos/Charts/PointsAggregationFinancialChart/description.md]:
```suggestion
The **Chart** widget can aggregate series points. To enable this feature, set the **aggregation**.[enabled](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/series/aggregation/#enabled) option of a [series](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/series/) to **true**. 
```

File: JSDemos/Demos/Charts/PointsAggregationFinancialChart/description.md [hunk 2]
```diff
@@ -1 +1,5 @@
...
\ No newline at end of file
+The **Chart** widget can aggregate data represented by series points. To enable this feature, set the **aggregation**.[enabled](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/series/aggregation/#enabled) option of a [series](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/series/) to **true**. 
+
+In this demo, the chart's [visual range](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/argumentAxis/visualRange/) depends on the [RangeSelector](/Documentation/ApiReference/Data_Visualization_Widgets/dxRangeSelector/)'s settings (see the [onValueChanged](/Documentation/ApiReference/Data_Visualization_Widgets/dxRangeSelector/Configuration/#onValueChanged) handler). When the selected range is reduced, the series automatically becomes less aggregated. To change this behavior and set the fixed aggregation degree, specify the [aggregationInterval](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/argumentAxis/aggregationInterval/) option of an axis.
```
Comment (RomanTsukanov) [inline JSDemos/Demos/Charts/PointsAggregationFinancialChart/description.md]: It is better to being from the beginning:

```suggestion
In this demo, the **Chart** is bound to the [RangeSelector](/Documentation/ApiReference/Data_Visualization_Widgets/dxRangeSelector/) component. When you narrow down the selection range, the chart zooms in, and the series points become less aggregated. The converse is also true: when the selected range is wider, the degree of aggregation is higher.
```

I also removed the last sentence for two reasons: 1) `aggregationInterval` is not the only means of controlling the degree of aggregation - there's also `aggregationGroupWidth`; 2) this demo has an option called `aggregationInterval` set for the RangeSelector, and it is not the same option. I think it is better to send the user to the `aggregation` description and let him find out the details from there. Hence, the suggestion for the paragraph below.

File: JSDemos/Demos/Charts/PointsAggregationFinancialChart/description.md [hunk 3]
```diff
@@ -1 +1,5 @@
...
+
+Refer to the [aggregation](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/series/aggregation/) option description to see possible options for different series types.
```
Comment (RomanTsukanov) [inline JSDemos/Demos/Charts/PointsAggregationFinancialChart/description.md]:
```suggestion
Aggregation on zoom is only one use case for this feature. Refer to the [aggregation](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/series/aggregation/) description for information on how to configure aggregation for other use cases.
```

**Review by RomanTsukanov — CHANGES_REQUESTED** [review]

File: JSDemos/Demos/Charts/PointsAggregationFinancialChart/description.md [hunk 4]
```diff
@@ -1 +1,5 @@
-This demo illustrates how the **Chart** widget aggregates series points in&nbsp;a&nbsp;financial chart. The **useAggregation** option activates this feature. The **RangeSelector** widget zooms the contents of&nbsp;the **Chart**. Note that whenever the selected range is&nbsp;reduced, the series becomes less aggregated.
\ No newline at end of file
+The **Chart** widget can aggregate series points. To enable this feature, set the **aggregation**.[enabled](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/series/aggregation/#enabled) option of a [series](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/series/) to **true**. 
+
+In this demo, the **Chart** is bound to the [RangeSelector](/Documentation/ApiReference/Data_Visualization_Widgets/dxRangeSelector/) component. When you narrow down the selection range, the chart zooms in, and the series points become less aggregated. The converse is also true: when the selected range is wider, the degree of aggregation is higher.
```
Comment (pavel-2020) [inline JSDemos/Demos/Charts/PointsAggregationFinancialChart/description.md]:
```suggestion
In this demo, the **Chart** is bound to the [RangeSelector](/Documentation/ApiReference/Data_Visualization_Widgets/dxRangeSelector/) component. The more the width of its selection range is, the bigger the graph scale and the degree of aggregation are, and vice versa. 
```

File: JSDemos/Demos/Charts/PointsAggregationFinancialChart/description.md [hunk 5]
```diff
@@ -1 +1,5 @@
...
+
+Aggregation on zoom is only one use case for this feature. Refer to the [aggregation](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/series/aggregation/) description for information on how to configure aggregation for other use cases.
```
Comment (pavel-2020) [inline JSDemos/Demos/Charts/PointsAggregationFinancialChart/description.md]:
```suggestion
Aggregation on zoom is only one use case for this feature. Refer to the [aggregation](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/series/aggregation/) description on how to configure aggregation for other use cases.
```

File: JSDemos/Demos/Charts/PointsAggregationFinancialChart/description.md (hunk 4 above)
Comment (Abadzhev) [inline JSDemos/Demos/Charts/PointsAggregationFinancialChart/description.md]: Just tightening up the other suggestion - it sound a bit better to me. 

```suggestion
In this demo, the **Chart** is bound to the [RangeSelector](/Documentation/ApiReference/Data_Visualization_Widgets/dxRangeSelector/) component. The wider you set the selection range, the higher aggregation degree the control uses, and vice versa.
```

File: JSDemos/Demos/Charts/PointsAggregationFinancialChart/description.md (hunk 5 above)
Comment (Abadzhev) [inline JSDemos/Demos/Charts/PointsAggregationFinancialChart/description.md]: I didn't want to repeat "cases" in two sentences. "One of many possible" sounds a bit better to me than "only one". The latter is somewhat vague - do we mean "one of many" or "the only one"?

```suggestion
Aggregation on zoom is one of many possible use cases for this feature. Refer to the [aggregation](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/series/aggregation/) description for information on how to configure aggregation for other scenarios.
```

File: JSDemos/Demos/Charts/PointsAggregationFinancialChart/description.md [hunk 6]
```diff
@@ -1 +1,5 @@
-This demo illustrates how the **Chart** widget aggregates series points in&nbsp;a&nbsp;financial chart. The **useAggregation** option activates this feature. The **RangeSelector** widget zooms the contents of&nbsp;the **Chart**. Note that whenever the selected range is&nbsp;reduced, the series becomes less aggregated.
\ No newline at end of file
+The **Chart** widget can aggregate series points. To enable this feature, set the **aggregation**.[enabled](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/series/aggregation/#enabled) option of a [series](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/series/) to **true**. 
+
+In this demo, the **Chart** is bound to the [RangeSelector](/Documentation/ApiReference/Data_Visualization_Widgets/dxRangeSelector/) component. The wider you set the selection range, the higher aggregation degree the control uses, and vice versa.
```
Comment (albertov05) [inline JSDemos/Demos/Charts/PointsAggregationFinancialChart/description.md]: In the demo, are you setting the "selection" to a wide range? For example, do you mean the following:

```suggestion
In this demo, the **Chart** is bound to the [RangeSelector](/Documentation/ApiReference/Data_Visualization_Widgets/dxRangeSelector/) component. If you set the selection to a wide range, the control will use a higher aggregation degree, and vice versa.
```

File: JSDemos/Demos/Charts/PointsAggregationFinancialChart/description.md [hunk 7]
```diff
@@ -1 +1,5 @@
...
+
+Aggregation on zoom is one of many possible use cases for this feature. Refer to the [aggregation](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/series/aggregation/) description for information on how to configure aggregation for other scenarios.
```
Comment (albertov05) [inline JSDemos/Demos/Charts/PointsAggregationFinancialChart/description.md]:
```suggestion
Aggregation on zoom is one of many possible use cases for this feature. For information on how to configure aggregation for other scenarios, refer to the following description: [aggregation](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/series/aggregation/).
```

**Review by RomanTsukanov — APPROVED** [review]

**Review by pavel-2020 — APPROVED** [review]

**Pull request merged**